### PR TITLE
Update reward services and fix bug

### PR DESCRIPTION
**Pull request by tintinthong**
### Description

- [x] Completes #CS-4409

### CODE REVIEW

**Review by paulinhapenedo — APPROVED** [review]
LGTM!

**Review by oidoug — APPROVED** [review]

File: cardstack/src/screens/RewardsCenterScreen/useRewardsDataFetch.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { networkTypes } from '@rainbow-me/helpers/networkTypes';
 import { useAccountSettings } from '@rainbow-me/hooks';
 
 const rewardDefaultProgramId = {
-  [networkTypes.sokol]: '0x5E4E148baae93424B969a0Ea67FF54c315248BbA',
+  [networkTypes.sokol]: '0x0885ce31D73b63b0Fcb1158bf37eCeaD8Ff0fC72',
   // TestID
```
Comment (danibonilha) [inline cardstack/src/screens/RewardsCenterScreen/useRewardsDataFetch.ts]: I know it's not related but, can you remove this TestID comment ? It's just a leftover  and it seems the xdai one is the test

File: cardstack/src/services/rewards-center/rewards-center-service.ts [hunk 2]
```diff
@@ -63,6 +63,7 @@ const getValidProofs = async ({
 
   const proofs = await rewardPoolInstance.getProofs(
     accountAddress,
+    undefined,
     rewardProgramId,
     tokenAddress,
```
Comment (danibonilha) [inline cardstack/src/services/rewards-center/rewards-center-service.ts:68]: I thought we were going to remove the getProofs entirely from the wallet, is this like the first step ? can you elaborate why this change is needed before migrating the functions ?

Comment (danibonilha) [reply cardstack/src/services/rewards-center/rewards-center-service.ts:68]: Oh is it just to update the testnet id ?

Comment (tintinthong) [reply cardstack/src/services/rewards-center/rewards-center-service.ts:68]: This is like the first step. Sorry I had alot of discussion with doug on how to restructure it. But theres another PR that I will have out for the sdk again. But I just didn want to break it based off the previous PR of `claimAll` to me that was not all encompassing bcos I didn consider rewardTOkenBalances but I think `getProofs` will remain as the change.

File: cardstack/src/screens/RewardsCenterScreen/useRewardsDataFetch.ts (hunk 1 above)
Comment (tintinthong) [reply cardstack/src/screens/RewardsCenterScreen/useRewardsDataFetch.ts]: Ok.

File: cardstack/src/services/rewards-center/rewards-center-service.ts (hunk 2 above)
Comment (tintinthong) [reply cardstack/src/services/rewards-center/rewards-center-service.ts:68]: This change just prevent the brokage

**Comment by tintinthong** [comment]
@danibonilha do I just merge this?

**Review by danibonilha — APPROVED** [review]

**Comment by tintinthong** [comment]
@danibonilha I updated to 1.0.11 sdk version which he just released. Do I merge this in?
